Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/SamplePaywallsList.swift

```diff
@@ -215,8 +215,6 @@ extension SamplePaywallsList {
         switch action {
         case .restoreCompleted(_):
             print("CustomerCenter: restoreCompleted")
-        case .purchaseCompleted(_):
-            print("CustomerCenter: purchaseCompleted")
```

Comment:
This was a leftover from a last minute change in #4057 which was causing paywall tester not to build.